tests/external/iree-test-suites/torch_models/sdxl/punet_benchmark_mi325_v2.json

```diff
@@ -0,0 +1,38 @@
+{
+  "type": "benchmark",
+  "markers": [
+    "hip",
+    "mi325",
+    "punet"
+  ],
+  "modules": [
+    "sdxl/modules/punet_gfx942_v2"
+  ],
+  "weights": [
+    {
+      "type": "url",
+      "url": "https://sharkpublic.blob.core.windows.net/sharkpublic/iree-test-suites/torch-models/sdxl/punet/punet_int8.irpa",
+      "scope": "model"
+    }
+  ],
+  "run_args": [
+    "--hip_use_streams=true",
+    "--hip_allow_inline_execution=true",
+    "--device_allocator=caching",
+    "--function=run_forward",
+    "--device=hip",
+    "--benchmark_repetitions=10",
+    "--benchmark_min_warmup_time=3.0"
+  ],
+  "inputs": [
+    { "value": "1x4x128x128xf16" },
+    { "value": "2x64x2048xf16" },
+    { "value": "2x1280xf16" },
+    { "value": "2x6xf16" },
+    { "value": "1xf16" },
+    { "value": "1xsi64" },
+    { "value": "100xf32" },
+    { "value": "100xf32" }
+  ],
+  "golden_time_ms": 43.5
```

Comment:
> 10% is the usual max delta across chips from the same SKU

I recently learned that we should set it to actual_time * 1.1, which is 40.263	* 1.1 ~= 44.28. Can you make the change?